ihatemoney/models.py

```diff
@@ -51,14 +51,25 @@
     ],
 )
 
+
 class BillType(Enum):
```

Comment:
There is a `FormEnum` helper here https://github.com/spiral-project/ihatemoney/blob/4af4c10b1fe7781207a7634823e200df40146935/ihatemoney/utils.py#L324-L344 but we can't use it since it expects int values for the extending class.

Reply:
Indeed I also found this while adding the enum, but I wanted to have an Enum with string values to easily show a "nice" string to the user.